### DISCUSSION TITLE
Can anyone tell me why these keybinds don't work at all? (Print key with grim)

### DISCUSSION THREAD

**Discussion by charbelnicolas** [labwc/labwc#1421] [Q&A] [answered]
I have the following two keybinds which I can't make work with labwc... If you have any idea why that is please let me know, I'd really appreciate the help.

```
<keybind key="Print">
    <action name="Execute" command="grim -t png -l 0 ~/Screenshots/ss_$(date +%Y_%m_%d)_$(date +%H_%M_%S).png" />
</keybind>

<keybind key="S-Print">
    <action name="Execute" command='grim -t png -l 0 -g "$(slurp)" ~/Screenshots/ss_$(date +%Y_%m_%d)_$(date +%H_%M_%S).png' />
</keybind>
```

Thank you

**Comment by Flrian** [comment, marked as answer]
I'm pretty sure your quotes won't work if you write the keybind like that, [take a look](https://labwc.github.io/labwc-config.5.html#syntax) at labwc documentation. Then the second issue is that you aren't running the command in a shell, you are just executing grim. There is an example in the site I linked, or here is mine:

```
    <keybind key="Print">
      <action name="Execute">
        <command>sh -c 'grim $(xdg-user-dir PICTURES)/$(date +%Y-%m-%dT%H:%M:%S)_grim.png'</command>
      </action>
    </keybind>
```

**Reply by charbelnicolas** [reply]
Thank you!
